test/setupTests.test.js

```diff
@@ -0,0 +1,7 @@
+const sinon = require('sinon');
+
+before(() => {
```

Comment:
would be nice to have in-code explanation for the purpose, or you can use magic of naming and have `setupNoiseReduction.js` or similars ;)

Reply:
I've added a comment about this stub. Will leave filename to be used as single source of global test configurations.